sci-mathematics/axel/axel-2017.02.16.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils user
+
+DESCRIPTION="Algebraic geometric modeling platform"
+HOMEPAGE="http://dtk.inria.fr/axel/"
+SRC_URI="https://timeraider4u.github.io/distfiles/files/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="~sci-libs/dtk-${PV}"
+DEPEND="${RDEPEND}"
+
+AXEL_DATA_DIR="/usr/share/axel/data"
+AXEL_PLUGINS_DIR="/usr/lib/axel-plugins"
+AXEL_GROUP="dtk-axel"
```

Comment:
Please move those definitions into `pkg_setup` so that the global space doesn't get polluted.